src/presentation/mesh-tag-formatter.ts

```diff
@@ -0,0 +1,16 @@
+import { MeshTag } from "../mesh/mesh-tenant.model.ts";
+import { MeshTableTag } from "./mesh-table.ts";
+import { dim, Table, yellow } from "../deps.ts";
+
+export function formatMeshTags(meshTags: MeshTag[]) {
```

Comment:
d: this function is only used in the `MeshTenantTableViewGenerator` as well as the `MeshTenantCostTableViewGenerator`, it makes sense to not mix imperative programming when the OOP parts are there. I would rather suggest turning the TableGenerator into an abstract class and place this code there as a protected method to not have it "dangling" here as it rather is hard tied to this classes.